src/lib/fields/Field.js

```diff
@@ -28,6 +28,27 @@ export class Field {
     return record;
   }
 
+  /**
+   * Deserialize backend creatibutor errors into format compatible with frontend.
+   * @method
+   * @param {array} errors - array of error objects
+   * @returns {object} - object representing errors
+   */
+  deserializeErrors(errors) {
+    let deserializedErrors = {};
+
+    // TODO - WARNING: This doesn't convert backend error paths to frontend
+    //                 error paths. Doing so is non-trivial
+    //                 (re-using deserialize has some caveats)
+    //                 Form/Error UX is tackled in next sprint and this is good
+    //                 enough for now.
+    for (let e of errors) {
+      _set(deserializedErrors, e.field, e.messages.join(' '));
+    }
+
+    return deserializedErrors;
+  }
+
```

Comment:
I don't think we need this, or do we?

Reply:
I think "maybe" since we are deserializing schemas, so errors have correct field paths... but let's remove and see in the future. We need to put some more work on error de/serialization